server/lib/wsapi/lobbies.js

```diff
@@ -143,7 +143,7 @@ export class LobbyApi {
     }),
   )
   async create(data, next) {
-    const { name, map, gameType, gameSubType } = data.get('body')
+    const { name, map, gameType, gameSubType, allowObservers } = data.get('body')
```

Comment:
Yeah, I was thinking our (new and improved!) create-lobby-ui should have a checkbox to enable this.

I guess we could make it `true` by default on server?